### DISCUSSION TITLE
Enable follow-redirect option `trackRedirects` in Axios

### DISCUSSION THREAD

**Discussion by saqib-ahmed** [axios/axios#6487] [Q&A]
Is it possible to enable the `trackRedirects` flag provided by `follow-redirects` in Axios. I couldn't find any mention in the code search or documentation. 

My use case is that I want to track all the intermediate redirect steps while resolving a URL. I can see this [code here](https://github.com/follow-redirects/follow-redirects/blob/35a517c5861d79dc8bff7db8626013d20b711b06/index.js#L377) in follow-redirects which adds the list of all the intermediate URLs. I can see this object in axios response, but it is empty because `trackRedirects` is `false` by default.

**Comment by jasonsaayman** [comment]
Closing as I doubt we will be doing this anytime soon.
